src/assets/stylesheets/presence-log.scss

```diff
@@ -35,6 +64,29 @@
       max-width: 75%;
     }
 
+    :local(.spawn-message) {
```

Comment:
IN my branch I split this off into an `icon-butotn` class. We should also technically probably do the hover effect in CSS instead of needing 2 images, especially since we don't spritesheet them